src/extensions/timeseries/TimeseriesViewNew/useTimeseriesModel.ts

```diff
@@ -3,8 +3,8 @@ import { createCalculationPool, HitherContext, HitherInterface } from "../../com
 import useFetchCache from "../../common/useFetchCache"
 import { RecordingInfo } from "../../extensionInterface"
 
-// it may be important to limit this to 1 at a time when using a filter
-const timeseriesCalculationPool = createCalculationPool({maxSimultaneous: 1, method: 'stack'})
+// it may be important to limit this when using a filter
+const timeseriesCalculationPool = createCalculationPool({maxSimultaneous: 4, method: 'stack'})
```

Comment:
Increase maxSimultaneous here from 1 to 4... because now we know that the number of worker threads is limited to 1 for each bandpass filter process.